### DISCUSSION TITLE
Use system dependencies

### DISCUSSION THREAD

**Discussion by oraqlle** [cabinpkg/cabin#777] [Q&A] [answered]
Is there a way to specify the include path of system installed packages?

**Comment by ken-matsui** [comment]
@oraqlle Are they not included?

**Reply by oraqlle** [reply]
@ken-matsui They would be is I could use a different compiler (GCC-13.1 from Homebrew vs GCC-11.3 from system). Some core Conan packages (OpenCV), refuse to build due to some dependency issues (even in a Conan package) but as a work-around I wanted to use a library I installed from Homebrew but its not in the default path of the system compiler.

**Comment by ken-matsui** [comment, marked as answer]
This is implemented in 0.8.0.
